src/vivarium/framework/event.py

```diff
@@ -41,15 +41,15 @@ class Event(NamedTuple):
 
     Attributes
     ----------
-    index :
+    index 
```

Comment:
just fyi if you're using type annotations and so not including the types in the docstring, you don't put the colons
see e.g., https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_numpy.html